exercism/largest-series-product/largest-series-product.js

```diff
@@ -0,0 +1,30 @@
+var Series = function(inputNumber) {
+  if( /[^0-9]/g.test(inputNumber) ) {
```

Comment:
I've noticed this nice errors catchers in all the right places in the code. Great! Thanks for teaching me this.